### PR TITLE
OSDOCS#6464: Added bug fix for parse image reference for oc-mirror in the release note

### PR DESCRIPTION
**Pull request by subhtk**
Version(s): 4.14

Issue: [OSDOCS-6464](https://issues.redhat.com/browse/OSDOCS-6464)

Link to docs preview:
[Doc Preview](https://61802--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes.html#ocp-4-14-bug-fixes) OpenShift CLI (oc) section in bug fixes.

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by sherine-k — APPROVED** [review]
Thank you @subhtk !

File: release_notes/ocp-4-14-release-notes.adoc [hunk 1]
```diff
@@ -496,6 +496,14 @@ In the following tables, features are marked with the following statuses:
 [id="ocp-4-14-openshift-cli-bug-fixes"]
 ==== OpenShift CLI (oc)
 
+Previously, container image references that have both tag and digest were not correctly interpreted by oc-mirror and resulted in an error message like below:
```
Comment (eohartman) [inline release_notes/ocp-4-14-release-notes.adoc:499]: Just a minor suggestion to change this to say "...error message like the one below:"

Comment (subhtk) [reply release_notes/ocp-4-14-release-notes.adoc:499]: Ack.

File: release_notes/ocp-4-14-release-notes.adoc [hunk 2]
```diff
@@ -496,6 +496,14 @@ In the following tables, features are marked with the following statuses:
 [id="ocp-4-14-openshift-cli-bug-fixes"]
 ==== OpenShift CLI (oc)
 
+Previously, container image references that have both tag and digest were not correctly interpreted by oc-mirror and resulted in an error message like the one below:
```
Comment (kelbrown20) [inline release_notes/ocp-4-14-release-notes.adoc:499]:
```suggestion
Previously, container image references that have both tag and digest were not correctly interpreted by the oc-mirror plug-in and resulted in the following error:
```

File: release_notes/ocp-4-14-release-notes.adoc [hunk 3]
```diff
@@ -496,6 +496,14 @@ In the following tables, features are marked with the following statuses:
...
+
+[source,yaml]
+----
+"localhost:6000/cp/cpd/postgresql:13.7@sha256" is not a valid image reference: invalid reference format
+----
+
+This behavior has been fixed and such references are now accepted and correctly mirrored.
```
Comment (kelbrown20) [inline release_notes/ocp-4-14-release-notes.adoc:506]:
```suggestion
This behavior has been fixed, and the references are now accepted and correctly mirrored. 
```

**Review by kelbrown20 — COMMENTED** [review]
Hello! Just some wording suggestions to consider before merging. Thank you!

File: release_notes/ocp-4-14-release-notes.adoc (hunk 2 above)
Comment (subhtk) [reply release_notes/ocp-4-14-release-notes.adoc:499]: Ack.

File: release_notes/ocp-4-14-release-notes.adoc (hunk 3 above)
Comment (subhtk) [reply release_notes/ocp-4-14-release-notes.adoc:506]: ck.

**Comment by subhtk** [comment]
@kelbrown20 Done with the changes, please go ahead with the merge.

**Review by kelbrown20 — COMMENTED** [review]
Updates LGTM! Great work!

**Pull request merged**